app/plugin_api.go

```diff
@@ -98,6 +99,10 @@ func (api *PluginAPI) SavePluginConfig(pluginConfig map[string]interface{}) *mod
 	return api.app.SaveConfig(cfg, true)
 }
 
+func (api *PluginAPI) GetBundlePath() string {
```

Comment:
1/5 I'd be supportive of all APIs reserving an error out parameter, for future compatibility if nothing else